### ISSUE TITLE
Typo: Animation doc, reduce-motion should be reduced-motion

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#18751]
**Describe the bug**
Within the [Accessibility Considerations](https://github.com/WordPress/gutenberg/blob/bc8adf7378255c0ed54d72b699c4c74bb2968704/docs/designers-developers/designers/animation.md#accessibility-considerations) section in the Animation doc, we refer to the `prefers-reduce-motion` media query, which should be `prefers-reduced-motion` - as [the query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) is used.

> Whenever possible, ensure that animations respect the OS-level "Reduce Motion" settings. This can be done by utilizing the prefers-reduce-motion media query. Gutenberg includes a @reduce-motion mixin for this, to be used alongside rules that include a CSS animate property.

**Expected behavior**
Simply, `prefers-reduce-motion` should be `prefers-reduced-motion`.
